src/components/ui/QuantitySelector/QuantitySelector.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react'
+import { QuantitySelector as UIQuantitySelector } from '@faststore/ui'
+import { Plus, Minus } from 'phosphor-react'
+
+import './quantity-selector.scss'
+
+interface QuantitySelectorProps {
+  maxQuantity: number
+  minQuantity: number
+  inputDisabled: boolean
```

Comment:
Can't we just use `disabled` here instead of `inputDisabled `?

Reply:
@hellofanny, fixed at https://github.com/vtex-sites/base.store/pull/183/commits/10c248ac096653802891832183f6cfa2d8344fdf

Reply:
@eduardoformiga fixed at https://github.com/vtex-sites/base.store/pull/183/commits/5a252ea8cc927220709bb298ad9ba6cb65c7438f

Reply:
Thanks for the suggestions. It is looking way cleaner now. 😀